spec/index.bs

```diff
@@ -7364,14 +7380,40 @@ rendering operations have been flushed to the drawing buffer.
         that created this swap chain next.
 
         <div algorithm="GPUSwapChain.getCurrentTexture">
-            **Called on:** {{GPUSwapChain}} this.
+            **Called on:** {{GPUSwapChain}} |this|.
 
             **Returns:** {{GPUTexture}}
 
-            Issue: Describe {{GPUSwapChain/getCurrentTexture()}} algorithm steps.
+            1. Set |this|.{{GPUSwapChain/[[dirty]]}} to `true`.
+            1. Return |this|.{{GPUSwapChain/[[currentTexture]]}}.
         </div>
 </dl>
 
+<div algorithm>
+    During the "update the rendering [of the] `Document`" step of the "[=Update the rendering=]"
+    HTML processing model, the associated {{HTMLCanvasElement}} |canvas| for each {{GPUSwapChain}}
+    |swapChain| should <dfn abstract-op>present the swap chain content to the canvas</dfn> by running
+    the following steps:
+
+        1. If |swapChain|.{{GPUSwapChain/[[dirty]]}} is `false`, stop.
+        1. Let |displayTexture| be |swapChain|.{{GPUSwapChain/[[currentTexture]]}}.
+        1. Let |swapChain|.{{GPUSwapChain/[[currentTexture]]}} be a new {{GPUTexture}} with a
```

Comment:
Maybe we could define the descriptor that's used because right now we don't define miplevels, the depth, sampleCount, etc.